lib/kitchen/provisioner/dependencies.erb

```diff
@@ -55,34 +58,38 @@ def install_dependencies
   end
 
   # install formulas
-  config[:dependencies].select{|dependency| dependency.has_key?(:repo)}.each do |formula|
+  config[:dependencies].each do |formula|
     #unless config[:vendor_repo].has_key?(formula[:repo])
     #  raise UserError, "kitchen-salt: Invalid dependency formula :repo, no such vendor_repo '#{formula[:repo]}' specified."
     #end
-    case formula[:repo]
-    when 'git'
-      script += <<-INSTALL
-        fetchGitFormula #{formula[:source]} "#{formula[:name]}" "#{formula[:branch] || 'master'}"
-      INSTALL
-    when 'spm'
-      # TODO: SPM should know the sandbox_path is /tmp/kitchen/etc
-      script += <<-INSTALL
-        #{sudo('spm')} install #{formula[:package]||formula[:name]};
-      INSTALL
-    when 'yum'
-      script += <<-INSTALL
-        #{sudo('yum')} install -y #{formula[:package]||formula[:name]};
-      INSTALL
-    when 'apt'
-      script += <<-INSTALL
-        #{sudo('apt-get')} install -y #{formula[:package]||formula[:name]};
-      INSTALL
+    if formula.key?(:repo)
```

Comment:
Why are you checking for key :repo when the dependencies example above does not use repo and instead uses package?